src/azure-cli/azure/cli/command_modules/appconfig/feature.py

```diff
@@ -142,17 +147,28 @@ def set_feature(cmd,
 
 
 def delete_feature(cmd,
-                   feature,
+                   feature=None,
+                   key=None,
                    name=None,
                    label=None,
                    yes=False,
                    connection_string=None,
                    auth_mode="key",
                    endpoint=None):
+    if key is None and feature is None:
+        raise CLIError("Please provide either `--key` or `--feature` value.")
```

Comment:
Could you please use the specific error type `RequiredArgumentMissingError` instead of `CLIError`